uber/models.py

```diff
@@ -700,13 +700,15 @@ def delete_from_group(self, attendee, group):
             self.delete(attendee)
             group.attendees.remove(attendee)
 
-        def assign_badges(self, group, new_badge_count, new_badge_type=None, **extra_create_args):
-            new_badge_type = new_badge_type or c.ATTENDEE_BADGE
```

Comment:
I only see these, I think we're ok:

![image](https://cloud.githubusercontent.com/assets/5413064/9199029/0743a5ee-400e-11e5-81ab-5ce6eba3f2d0.png)